plugins/scaffolder-backend/src/lib/files/serializeDirectoryContents.ts

```diff
@@ -44,19 +44,34 @@ export async function serializeDirectoryContents(
     dot: true,
     gitignore: options?.gitignore,
     followSymbolicLinks: false,
+    // In order to pick up 'broken' symlinks, we oxymoronically request files AND folders yet we filter out folders
+    // This is because broken symlinks aren't classed as files so we need to glob everything
+    onlyFiles: false,
     objectMode: true,
     stats: true,
   });
 
   const limiter = limiterFactory(10);
 
   return Promise.all(
-    paths.map(async ({ path, stats }) => ({
-      path,
-      content: await limiter(async () =>
-        fs.readFile(joinPath(sourcePath, path)),
-      ),
-      executable: isExecutable(stats?.mode),
-    })),
+    paths
+      .filter(({ dirent }) => !dirent.isDirectory())
+      .filter(({ dirent, path }) => {
+        if (!dirent.isSymbolicLink()) return true;
+        if (!fs.existsSync(resolveSafeChildPath(sourcePath, path))) return true; // We only want symlinks that DO NOT exist (yet)
+        return false;
+      })
+      .map(async ({ dirent, path, stats }) => ({
+        path,
+        content: await limiter(async () => {
+          const absFilePath = resolveSafeChildPath(sourcePath, path);
+          if (dirent.isSymbolicLink()) {
+            return fs.readlinkSync(absFilePath, 'buffer');
```

Comment:
this too

Reply:
Replaced with `fs.readlink` from `fs/promises`

Reply:
I'll also note that `fs.readlink` appears to work different in `fs-extra` than it does with `fs/promises`. From memory, the typing would complain but it was a couple days ago that I did it so I've mostly forgotten